documentation/asciidoc/topics/proc_securing_custom_configuration.adoc

```diff
@@ -0,0 +1,39 @@
+[id='secure_credentials_{context}']
+= Securing custom {brandname} configuration
+
+[role="_abstract"]
+Define and store custom {brandname} configuration that requires credentials securely.
```

Comment:
I don't think the last part of the sentence makes sense. How about something like:

"Securely define and store custom {brandname} server configurations"?